src/components/Input/Checkbox/story.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { boolean, text } from '@storybook/addon-knobs';
+
+import Checkbox from '.';
+import Readme from './README.md';
+
+import { Wrap } from '../../../stories/storybook-helpers';
+
+interface InteractiveInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  onChange: () => void;
+  InputRenderer: React.FC<React.InputHTMLAttributes<HTMLInputElement>>;
+}
+
+const InteractiveInput: React.FC<InteractiveInputProps> = ({
```

Comment:
Maybe I should promote this to a test util? I can see using it a lot when writing stories for input components.
This was copy-paste with mods from the original, in the `Input/story.tsx` file.